### ISSUE TITLE
Sorting items for the lowest price of each item

### ISSUE THREAD

**Issue by TheSorm** [Auctionator/Auctionator#1042]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when i dont know which items are currently selling for a high priece because if i sort for highest priece i only get items that are put into the action house for to high.

**Describe the solution you'd like**
I would like to have a list that shows the lowest costing item of each kind sorted for the priece, so that i can see which items are actually selling for a high price atm. 
An example with 4 items. 
Cloth for 2g
Cloth for 4g
Stone for 5g
Stone for 6g

List:
Stone (5g)
Cloth (2g)

**Describe alternatives you've considered**
Basically this list is in the add on allready. If the buy tab would allow me to search without any search input, the result would basically be what i want. That means that if the search could operate on scann data and would allow such a search, the feature is basically there allready.

**Comment by plusmouse** [comment]
You can already search with no input on the "Buy" tab - just click "Search".

Are you looking to detect price spikes as it doesn't sound like the feature as requested will detect that?

**Comment by plusmouse** [comment]
Are you on classic - the answer I gave assumes you're on retail?

**Comment by TheSorm** [comment]
Ah you can ok, i didnt noticed because it dosnt show anything while scanning first. Only the search button turns gray. That could maybe be improved with an indicator of how far the indexing or scanning is atm :)  (I assume it is doing something in the background?)

No i was just looking for a way to find out which items are worth farming atm because they have a high cheapest price atm. 
Thats not really possible with the standart search

**Comment by plusmouse** [comment]
OK. Closing this issue as making that work would require new features to be added to TBC Auctionator, but we're not looking to do so:
![image](https://user-images.githubusercontent.com/60280559/121958128-3af84800-cd5b-11eb-9226-98861defbce2.png)

You can try the trade goods search though, this version of Auctionator will let you run the search:
[Auctionator-2.5.2-1-g0262647-bcc.zip](https://github.com/Auctionator/Auctionator/files/6651225/Auctionator-2.5.2-1-g0262647-bcc.zip)
